src/totalpave/utilities.cpp

```diff
@@ -23,105 +23,9 @@
-                break;
-            case SQLITE_EMPTY:
-                error = "SQLITE_EMPTY: A code that you should not see.";
-                break;
-            case SQLITE_ERROR:
-                error = "SQLITE_ERROR: An unknown SQLite error occurred.";
-                break;
-            case SQLITE_INTERNAL:
-                error = "SQLITE_ERROR: An internal SQLite error occurred.";
-                break;
-            case SQLITE_PERM:
-                error = "SQLITE_PERM: Permission denied.";
-                break;
-            case SQLITE_LOCKED:
-                error = "SQLITE_LOCKED: Write operation cannot continue due to a lock from another operation.";
-                break;
-            case SQLITE_NOMEM:
-                error = "SQLITE_NOMEM: Could not allocate memory.";
-                break;
-            case SQLITE_READONLY:
-                error = "SQLITE_READONLY: Attempted to write on a readonly connection.";
-                break;
-            case SQLITE_INTERRUPT:
-                error = "SQLITE_INTERRUPT: Operation was interrupted.";
-                break;
-            case SQLITE_IOERR:
-                error = "SQLITE_IOERR: Underlying I/O error occurred.";
-                break;
-            case SQLITE_NOTFOUND:
-                error = "SQLITE_NOTFOUND: See SQLite documentation.";
-                break;
-            case SQLITE_FULL:
-                error = "SQLITE_FULL: Disk is full.";
-                break;
-            case SQLITE_PROTOCOL:
-                error = "SQLITE_PROTOCOL: Failed to file lock. Should try again.";
-                break;
-            case SQLITE_SCHEMA:
-                error = "SQLITE_SCHEMA: Schema has changed.";
-                break;
-            case SQLITE_TOOBIG:
-                error = "SQLITE_TOOBIG: Blob or SQL statement is too large.";
-                break;
-            case SQLITE_MISMATCH:
-                error = "SQLITE_MISMATCH: Data type mismatch.";
-                break;
-            case SQLITE_MISUSE:
-                error = "SQLITE_MISUSE: Application error.";
-                break;
-            case SQLITE_NOLFS:
-                error = "SQLITE_NOLFS: No Large File Support.";
-                break;
-            case SQLITE_AUTH:
-                error = "SQLITE_AUTH: SQL Statement not authorized.";
-                break;
-            case SQLITE_RANGE:
-                error = "SQLITE_RANGE: Bind/Column index is out of range.";
-                break;
-            case SQLITE_FORMAT:
-                error = "SQLITE_FORMAT: Appearently not used.";
-                break;
-            case SQLITE_NOTADB:
-                error = "SQLITE_NOTADB: DB does not appear to be a SQLite DB.";
-                break;
-            default:
-                error = "An uncaught SQLite error (" + std::to_string(code) + ") have been reached.";
-                break;
-        }
-
-        return error;
-    }
```

Comment:
Yes this is a "breaking change".  v1 isn't truly out so I dont think it matters.